### PR TITLE
Eng 1466 fix bug where csv upload modal doesnt show the full rows without scrolling

### PR DESCRIPTION
**Pull request by eunice-chan**
## Describe your changes and why you are making these changes
Show the full data table so we don't need to scroll through to see all the rows.

## Related issue number (if any)
ENG 1466 

## Loom demo (if any)
https://user-images.githubusercontent.com/30596854/192826902-d526b4b7-8506-4435-b14d-2557c69aead5.mov

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [N/A] If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/integrations/dialogs/IntegrationFileUploadField.tsx [hunk 1]
```diff
@@ -123,7 +123,7 @@ export const IntegrationFileUploadField: React.FC<
     const overlay = dragging && theme.palette.gray[100];
     const styling = {
       margin: '16px',
-      height: '16ch',
+      padding: '16px',
```
Comment (agiron123) [inline src/ui/common/src/components/integrations/dialogs/IntegrationFileUploadField.tsx:126]: out of curiosity, how tall is this field?

**Comment by cw75** [comment]
@eunice-chan reminder to rename the PR title to be a full sentence. :)

**Review by likawind — APPROVED** [review]
This change looks fine. I think we should file another task to make the header section follow the same color theme as our other data tables (cc @agiron123 )

File: src/ui/common/src/components/integrations/dialogs/IntegrationFileUploadField.tsx (hunk 1 above)
Comment (eunice-chan) [reply src/ui/common/src/components/integrations/dialogs/IntegrationFileUploadField.tsx:126]: This is the styling for the dotted box around the "Upload the CSV file" text.

image

**Comment by eunice-chan** [comment]
@likawind Good thinking! Made a task here: https://linear.app/aqueducthq/issue/ENG-1765/update-upload-csv-table-view-to-match-the-table-artifact-preview-view

This is a change to main so I don't think we need to make the change right now although that is something we would want to get done in the ui-redesign branch eventually.

**Pull request merged**